src/components/ActionButtonWithModal/index.jsx

```diff
@@ -16,7 +15,7 @@ class ActionButtonWithModal extends React.Component {
     return (
       <React.Fragment>
         <Button
-          className={classNames(buttonClassName, 'btn-link btn-sm p-0')}
+          className={buttonClassName}
```

Comment:
making this component less opinionated for styling and letting consumers of it have full control over the button's styles.